model/Core/Properties/definingDocument.md

```diff
@@ -4,15 +4,15 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-URI for an SPDX document which defines an SPDX element.
+Artifact containing SPDX data which defines one or more SPDX elements.
 
 ## Description
 
-A definingDocument property is used to link an Element identifier to an SpdxDocument which contains the definition for the Element.
+A definingDocument property is used to link an Element identifier to an Artifact which contains the definition for the Element.
```

Comment:
Suggest small tweak:

"A definingDocument property is used to link the Element identifier for an Element defined external to a given "XCollection" to an Artifact Element representing the SPDX serialization instance which contains the definition for the Element."

Reply:
Updated per suggestion